src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1415px;
+  margin: auto;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo__link {
+  line-height: 0;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-self: center;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  letter-spacing: 0;
+  line-height: 60px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: #00acdc;
```

Comment:
It is a good practice to make all colors variable

Reply:
fixed all the comments